Code/Framework/AzCore/AzCore/Math/Matrix3x4.inl

```diff
@@ -203,6 +203,17 @@ namespace AZ
     }
 
 
+    AZ_MATH_INLINE Matrix3x4 Matrix3x4::CreateFromMatrix4x4(const Matrix4x4& matrix4x4)
```

Comment:
A 4x4 matrix is not generally convertible to a 3x4 matrix (loses shear components). Should we rename this method or provide some indicator that this is, in fact, a lossy transformation?

Reply:
We can do, I went for this naming more out of consistency (looking at `Matrix3x3::CreateFromMatrix4x4` and other APIs that have functions called `CreateFromMatrix4x4`, e.g. `Quaternion`. What do you think would be an appropriate name? `CreateFromMatrix4x4Lossy` (with a comment?)

Reply:
Discussed offline - call is now called `UnsafeCreateFromMatrix4x4`